index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/1ca239d8ab.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Version Portfolio</title>
+</head>
+<body>
+  <div class="container">
+
+    <section class="container-section">
+    <header>
+      <nav class="nav-header">
+        <div class="logo">My Logo</div>
+        <span class="bars"><i class="fa-solid fa-bars"></i></span>
+      </nav>
```

Comment:
- Great job so far 👍 Your site looks really good and it follows the template design closely but there are still a few issues to be tackled 😄 
- [ ] First, only your hamburger icon should be in the `nav` tag. This is because it is the navigation menu of the mobile view of your website. So kindly remove the logo `div `from the `nav` tag 👍 
- [ ] Also, your logo should be wrapped in an `a` tag. This is because your logo is a link and as a result should be placed in an `a` tag. Kindly add this feature.